force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -191,10 +275,14 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer adjustOrderOfOperations1(Integer a, Integer b, Integer c) {
         // Add parentheses around the addition operation so that it is performed before multiplication
-        Integer result = a + b * c;
+        if (a==null || b==null || c==null) {
+            return null;
+        }
+
+        Integer result = (a + b) * c;
```

Comment:
Great job! This is a perfect example of a proper null check and then the rest of method logic.